From 188783e7580b7915e1f0fb78b0a894767c0392c7 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@kernel.org>
Date: Wed, 2 May 2018 09:28:33 -0500
Subject: [PATCH 08/11] clk: socfpga: stratix10: suppress unbinding platform's
 clock driver

commit a61315473a93e2d36bca276f9b03738936d94a1c upstream

The Stratix10 clock driver is essential to system operation, so their
removal should never happen.

Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Signed-off-by: Stephen Boyd <sboyd@kernel.org>
Signed-off-by: Tong.Chen <tong.chen@windriver.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/socfpga/clk-s10.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/socfpga/clk-s10.c b/drivers/clk/socfpga/clk-s10.c
index 1ffe3f0..7271463 100644
--- a/drivers/clk/socfpga/clk-s10.c
+++ b/drivers/clk/socfpga/clk-s10.c
@@ -329,6 +329,7 @@ static int s10_clkmgr_probe(struct platform_device *pdev)
 	.probe		= s10_clkmgr_probe,
 	.driver		= {
 		.name	= "stratix10-clkmgr",
+		.suppress_bind_attrs = true,
 		.of_match_table = stratix10_clkmgr_match_table,
 	},
 };
-- 
1.7.9.5

